### PR TITLE
Updated steps for creating TodosScreen.

### PR DESCRIPTION
**Pull request by EmperorYP7**
This PR is for resolving #4190 
Since this step was missing in the tutorial, I made appropriate modifications to passing-data.md file.
I'd be glad to know if there's something to add in here!

### CODE REVIEW

**Comment by sfshaza2** [comment]
@johnpryan or @domesticmouse, could one of you verify this?

**Comment by dev-aentgs** [comment]
> This PR is for resolving #4190
> Since this step was missing in the tutorial, I made appropriate modifications to passing-data.md file.
> I'd be glad to know if there's something to add in here!

In the first line remove space in between `Stateless Widget`

File: src/docs/cookbook/navigation/passing-data.md [hunk 1]
```diff
@@ -80,7 +80,48 @@ ListView.builder(
 So far, so good.
 This generates 20 todos and displays them in a ListView.
 
-## 3. Create a detail screen to display information about a todo
+## 3. Create a Todo screen to display the list
+
+For this, we create a `Stateless Widget`. We call it `TodosScreen`.
```
Comment (domesticmouse) [inline src/docs/cookbook/navigation/passing-data.md:85]: Per @dev-aentgs, please change `Stateless Widget` to `StatelessWidget`.

**Comment by domesticmouse** [comment]
@sfshaza2 sorry about the review delay!

**Comment by EmperorYP7** [comment]
@dev-aentgs and @domesticmouse sir, I hope all the issues are cleared now.

**Comment by domesticmouse** [comment]
LGTM, thanks @EmperorYP7 and @dev-aentgs =)

**Comment by EmperorYP7** [comment]
@sfshaza2 ma'am, please do review this PR once. I tried to clear all the issues in my previous commits.

**Comment by sfshaza2** [comment]
Thanks, all!

**Pull request merged**